src/components/Register.js

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import '../Register.css'
+import {register} from '../service/auth';
+import {Button, TextField} from "@material-ui/core";
+
+export class Register extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            login: '',
+            errorLoginText: '',
+            email: '',
+            emailPattern: /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
+            password: '',
+            repeatPassword: ''
+        };
+    }
+
+    onChange = (event) => {
+        console.log(event.target.name, event.target.value)
```

Comment:
I know that it is useful, but I suggest cleaning logging in MR. Just because if you are not alone on the project other people will have a lot of trash in the console)